crates/bevy_render/src/render_resource/resource_macros.rs

```diff
@@ -1,3 +1,12 @@
+#[cfg(target_has_atomic = "64")]
+pub type AtomicIdCounter = std::sync::atomic::AtomicU64;
```

Comment:
Can we just default to u32s? There's already a fairly low chance any of the IDs are going to overflow u32 to begin with.

Reply:
I guess so. If this turns out to be problematic we can change it back.